install/installer/pkg/common/common.go

```diff
@@ -446,28 +447,42 @@ func ConfigcatProxyEnv(ctx *RenderContext) []corev1.EnvVar {
 		if cfg.WebApp != nil && cfg.WebApp.ProxyConfig != nil && cfg.WebApp.ProxyConfig.Configcat != nil {
 			baseUrl = cfg.WebApp.ProxyConfig.Configcat.BaseUrl
 			pollInterval = cfg.WebApp.ProxyConfig.Configcat.PollInterval
+			fromConfigMap = cfg.WebApp.ProxyConfig.Configcat.FromConfigMap
 		}
 		return nil
 	})
 
 	if sdkKey == "" {
 		return nil
 	}
-
-	return []corev1.EnvVar{
+	envs := []corev1.EnvVar{
 		{
 			Name:  "CONFIGCAT_SDK_KEY",
 			Value: sdkKey,
 		},
-		{
-			Name:  "CONFIGCAT_BASE_URL",
-			Value: baseUrl,
-		},
-		{
-			Name:  "CONFIGCAT_POLL_INTERVAL",
-			Value: pollInterval,
-		},
 	}
+
+	if fromConfigMap != "" {
+		envs = append(envs,
+			corev1.EnvVar{
+				Name:  "CONFIGCAT_FROM_CONFIGMAP",
```

Comment:
I'd recommend changing this variable to be `CONFIGCAT_FILE=/path/to/file`. Not only is it more generic (does not assume knowledge of configmaps, which are k8s only concept) but it also means you can supply the path instead of hard-coding it.